src/test/java/CaseTests.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static core.utils.LoadEnvironmentFile.getDotenv;
+
+public class CaseTests {
+  private WebDriver driver;
+  private WebDriverWait wait;
+  private Map<String, Object> vars;
+  JavascriptExecutor js;
+
+  @BeforeTest
+  public void setUp() {
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--disable-notifications");
+    System.setProperty("webdriver.chrome.driver", getDotenv().get("CHROME_DRIVER_PATH"));
+    driver = new ChromeDriver(options);
+    wait = new WebDriverWait(driver, 30);
+    js = (JavascriptExecutor) driver;
+    vars = new HashMap<String, Object>();
+  }
+
+  @AfterTest
+  public void tearDown() {
+    driver.quit();
+  }
+
+  public void LoginWithValidCredentials() {
+    driver.get(getDotenv().get("BASE_URL_CASE"));
+    driver.findElement(By.id("username")).sendKeys(getDotenv().get("SALESFORCE_USERNAME"));
+    driver.findElement(By.id("password")).sendKeys(getDotenv().get("SALESFORCE_PASSWORD"));
+    driver.findElement(By.id("Login")).click();
+  }
+
+  public void logout() {
```

Comment:
There should be no logic in a test, so you should move all the methods (logout, clickOnElements, etc) to another file

Reply:
I will change that class to use the webDriverManager but I will work on that on my next PR.